validator/keymanager/remote-web3signer/client.go

```diff
@@ -0,0 +1,177 @@
+package remote_web3signer
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/crypto/bls"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+const (
+	ethApiNamespace = "/api/v1/eth2/sign/"
+	maxTimeout      = 5 * time.Second
```

Comment:
Should we have a shorter timeout? There are 12 seconds per slot and we wouldnt want to take up too much waiting for a hanging API response. I propose 3 seconds